spec/board_spec.rb

```diff
@@ -7,23 +7,25 @@
     @test_board = Board.new
   end
 
-  it 'an empty board has no marks' do
-    expect(@test_board.draw_board).to eq "\n 1 | 2 | 3 \n===+===+===\n 4 | 5 | 6 \n===+===+===\n 7 | 8 | 9 \n\n"
-  end
+#  it 'an empty board has no marks' do
+#    expect(@test_board.draw_board).to eq "\n 1 | 2 | 3 \n===+===+===\n 4 | 5 | 6 \n===+===+===\n 7 | 8 | 9 \n\n"
+#  end
```

Comment:
Just flagging comments to make sure you come back to this and either edit or remove this test :) given our discussion above, it might make sense to have a `format_board` function in the `Board` class and then you could use this test.